QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -257,11 +193,13 @@ private void Initialize()
         /// <param name="dataConfig">Subscription config to be removed</param>
         public void Unsubscribe(SubscriptionDataConfig dataConfig)
         {
+            Log.Trace($"DataBentoProvider.Unsubscribe(): Received unsubscription request for {dataConfig.Symbol}, Resolution={dataConfig.Resolution}, TickType={dataConfig.TickType}");
             _subscriptionConfigs.TryRemove(dataConfig.Symbol, out _);
```

Comment:
Remove it. It has extra logic.
Please use `_subscriptionManager`.
I have explained above.